### DISCUSSION TITLE
Antarctica boundary in a global mesh

### DISCUSSION THREAD

**Discussion by brey** [oceanmodeling/Management#13] [Q&A] [answered]
In a global mesh context what is the best way to represent Antarctica? The ice boundary (L5 in full resolution GSHHS) fits best with the DEMs. I presume that whether you have an ice model on/off would make a difference. Any thoughts @josephzhang8 @WPringle ?

**Comment by josephzhang8** [comment]
Hi George:

I checked my SMS files and I used L5 GHHS there. It's important to use RTopo2 to represent the bathymetry in Antarctica. I have not carefully tested the sensitivity to the boundary there. I did however test a simple ice shelf model (with atmos pressure acting as surrogate for ice shelf) using the data in RTopo/ice_base to represent the shelf height. The (short term) results seem to be close so I did not pursue further. That's a fascinating area to work in the future.

Regards,
-------------------------------
Joseph Zhang
(804)684 7595 (office)

SCHISM web: http://ccrm.vims.edu/schism/
From: George Breyiannis ***@***.***>
Sent: Sunday, February 5, 2023 6:59 AM
To: oceanmodeling/Management ***@***.***>
Cc: Y. Joseph Zhang ***@***.***>; Mention ***@***.***>
Subject: [Ocean Modeling Collaboration] Antarctica boundary in a global mesh (Discussion #13)


[EXTERNAL to VIMS received message]


In a global mesh context what is the best way to represent Antarctica? The ice boundary (L5 in full resolution GSHHS) fits best with the DEMs. I presume that whether you have an ice model on/off would make a difference. Any thoughts @josephzhang8<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fjosephzhang8&data=05%7C01%7Cyjzhang%40vims.edu%7C3cd72f5d179e488730e808db0770568c%7C8cbcddd9588d4e3b9c1e2367dbdf1740%7C0%7C0%7C638111951314261032%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=ssFICYQYOPoQ7ziJmt%2FjpTCTZMTxbc29WOk22ah91Pw%3D&reserved=0> @WPringle<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FWPringle&data=05%7C01%7Cyjzhang%40vims.edu%7C3cd72f5d179e488730e808db0770568c%7C8cbcddd9588d4e3b9c1e2367dbdf1740%7C0%7C0%7C638111951314261032%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Zo7N6%2B7cOvrutXNYHEQeY9rd8g5om8Z4d0GfC3w2FWc%3D&reserved=0> ?

-

**Comment by WPringle** [comment]
> In a global mesh context what is the best way to represent Antarctica? The ice boundary (L5 in full resolution GSHHS) fits best with the DEMs. I presume that whether you have an ice model on/off would make a difference. Any thoughts @josephzhang8 @WPringle ?

Use the GSHHS_f_L6 grounding-line boundary, not the ice boundary. Then you need to use the DEM giving the difference between the ice surface and the bedrock to get the depths under the ice. So can't just use GEBCO. See Section 2.4 of:
Pringle, W. J., Wirasaet, D., Roberts, K. J., & Westerink, J. (2021). Global Storm Tide Modeling with ADCIRC v55: Unstructured Mesh Design and Performance. Geoscientific Model Development, 14(2), 1125–1145. https://doi.org/10.5194/gmd-14-1125-2021

**Comment by brey** [comment]
Which one of the files [here](https://doi.pangaea.de/10.1594/PANGAEA.856844?format=html#download) should I use as the DEM?

**Comment by josephzhang8** [comment]
I used:

RTopo-2.0.1_30sec_Antarctica_data.nc


Regards,
-------------------------------
Joseph Zhang
(804)684 7595 (office)

SCHISM web: http://ccrm.vims.edu/schism/

**Comment by brey** [comment]
In the RTopo-2.0.1_30sec_Antarctica_data.nc there are several variables:

```python
<bound method DatasetAggregations.var of <xarray.Dataset>
Dimensions:              (londim: 43201, latdim: 4801)
Coordinates:
    lon                  (londim) float32 ...
    lat                  (latdim) float32 ...
Dimensions without coordinates: londim, latdim
Data variables:
    bedrock_topography   (latdim, londim) float32 ...
    ice_base_topography  (latdim, londim) float32 ...
    surface_elevation    (latdim, londim) float32 ...
Attributes:
    Title:        RTopo-2 global topography data set, Antarctic sector
    Author:       Janin Schaffer and Ralph Timmermann
    Institution:  Alfred Wegener Institute, Helmholtz Centre for Polar and Ma...
    Source:       Compilation of several original or gridded data sets
    References:   Schaffer, J., R. Timmermann, J.E. Arndt, S.S. Kristensen C....
    Version:      2.0
    Comment:      Use ncview -no_auto_overlay for a simple and efficient data...>
```

I assume that we use the `bedrock_topography`. True?

**Comment by WPringle** [comment, marked as answer]
@brey I use `ice_base_topography` minus `bedrock_topography` to get the depths under ice shelves. When `ice_base_topography` is zero/no ice then it will be `bedrock_topography`

**Comment by pmav99** [comment]
For the record, there is a newer dataset for under ice bathymetry: [paper](https://www.nature.com/articles/s41598-024-81599-1) and [data](https://datadryad.org/dataset/doi:10.5061/dryad.rbnzs7hkc)

They provide the water depth directly so you don't need to bother with multiple files. The relevant product is [WATER_HEIGHT_ANTGG2022.tif](https://datadryad.org/downloads/file_stream/3664028)

On average it gives larger depths for under-ice. Not sure if/how this translates to a measurable difference on a global model. If we find the time we should run a test to compare it with RTopo
